packages/amplify-category-analytics/amplify-plugin-index.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
```

Comment:
yes, it will be merged into the index.js. 
This was the implementation strategy we outlined before, so there's little interference between the development of the new plugin platform and other CLI developments.